src/lightning/fabric/connector.py

```diff
@@ -460,6 +461,16 @@ def _check_and_init_precision(self) -> Precision:
             return DeepSpeedPrecision(self._precision_input)  # type: ignore
         if isinstance(self.strategy, FSDPStrategy):
             return FSDPPrecision(precision=self._precision_input)  # type: ignore[arg-type]
+        if isinstance(self.strategy, ModelParallelStrategy) and self._precision_input not in (
+            "32-true",
+            "bf16-mixed",
+            "bf16-true",
+            "16-true",
+        ):
+            raise ValueError(
+                f"The `ModelParallelStrategy` does not support `Fabric(..., precision={self._precision_input!r})`."
+                " Choose a different precision."
```

Comment:
Can we specify what precisions are supported? Like "Choose a different precision among ..."

Reply:
Yes, good idea!